data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -6777,15 +6777,15 @@ spec:
                       instance during cluster creation.
                     type: string
                   tgName:
-                    description: TGName is the name of a pre-created TransitGateway
-                      inside IBM Cloud.
+                    description: TransitGateway is the name or id of a pre-created
+                      TransitGateway inside IBM Cloud.
                     type: string
                   userID:
                     description: UserID is the login for the user's IBM Cloud account.
                     type: string
                   vpcName:
-                    description: VPCName is the name of a pre-created VPC inside IBM
-                      Cloud.
+                    description: VPC is the name or id of a pre-created VPC inside
```

Comment:
`description` should start with `VPCName`

Reply:
can we?